src/linearmixedmodel.jl

```diff
@@ -574,14 +590,7 @@ function mkparmap(reterms::Vector{AbstractReMat{T}}) where {T}
     parmap
 end
 
-function StatsBase.modelmatrix(m::LinearMixedModel)
-    fe = first(m.feterms)
-    if fe.rank == size(fe, 2)
-        fe.x
-    else
-        fe.x[:, invperm(fe.piv)]
-    end
-end
+StatsBase.modelmatrix(m::LinearMixedModel) = m.feterm.x
```

Comment:
I've lost the thread a bit here: but the matrix is no longer stored permuted and is instead dynamically permuted as a view, correct?